docs/data_datacube.md

```diff
@@ -0,0 +1,57 @@
+# Creating datacubes
+
+## How to create a datacube
+
+TODO:
+- Describe how the `datacube.py` script works for a single MGRS tile
```

Comment:
@yellowcap, do you want to add a few paragraphs on how to run the `datacube.py` script for a single MGRS tile and date range?

Reply:
done